registration/models.py

```diff
@@ -0,0 +1,7 @@
+from django.db import models
+
+
+class Dashboard(models.Model):
```

Comment:
...?

Reply:
for the events, the user will register for